openerp/addons/base/ir/ir_fields.py

```diff
@@ -38,6 +38,19 @@ class ir_fields_converter(models.Model):
     _name = 'ir.fields.converter'
 
     @api.model
+    def _raise_import_error(self, error_type, error_msg, error_params=(), error_args=None):
```

Comment:
given that it doesn't `raise` anymore, maybe the method could be renamed just `_import_error` or `_build_import_error`? :-)